drivers/CoreSPI/tests/CoreSPI_test.cpp

```diff
@@ -0,0 +1,74 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "CoreSPI.h"
+
+#include "gtest/gtest.h"
+#include "stubs/mbed/SPI.h"
+
+using namespace leka;
+
+class CoreSPITests : public ::testing::Test
+{
+  protected:
+	CoreSPITests() : corespi(spi) {}
+
+	// void SetUp() override {}
+	// void TearDown() override {}
+
+	mbed::SPI spi {NC, NC, NC, NC};
+	CoreSPI corespi;
+
+	std::vector<uint8_t> expected_written_values {};
+};
+
+TEST_F(CoreSPITests, initialisation)
+{
+	ASSERT_NE(&corespi, nullptr);
+}
+
+TEST_F(CoreSPITests, writeOneByte)
+{
+	expected_written_values = {0x01};
```

Comment:
Je recommande plutôt de le créer ici plutôt que dans la classe un peu plus haut

```suggestion
	auto expected_written_values = std::vector<uint8_t> {0x01};
```

Ca prend pas plus de place et c'est plus lisible d'une part quand on lit l'entête des classe et pour identifier rapidement quel type on envoie en test